exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/SQLExpressionBuilder.kt

```diff
@@ -90,6 +90,68 @@ fun <T : Any?> ExpressionWithColumnType<T>.varPop(scale: Int = 2): VarPop<T> = V
  */
 fun <T : Any?> ExpressionWithColumnType<T>.varSamp(scale: Int = 2): VarSamp<T> = VarSamp(this, scale)
 
+// Window Functions
+
+/** Returns the number of the current row within its partition, counting from 1. */
+fun rowNumber(): RowNumber = RowNumber()
```

Comment:
Please avoid adding top-level functions for API-specific stuff. They will appear in code completion for every application with Exposed dependency

Reply:
In last commit I moved new window functions to ISqlExpressionBuilder. I should mention that some existing functions are also defined as top-level though and some are in ISqlExpressionBuilder. I didn`t grasp initial intention of this separation.